app/views/user_mailer/already_confirmed.html.slim

```diff
@@ -0,0 +1,14 @@
+html
+  head
+    meta content='text/html; charset=UTF-8' http-equiv='Content-Type'
+  body
+
+    p= 'A request was made to resend confirmation instructions to this email.' \
```

Comment:
Good point. We should probably be consistent about this. I'll checks the docs to see what the convention is.